src/sentry/static/sentry/less/errors.less

```diff
@@ -0,0 +1,38 @@
+.detailed-error {
+  display: flex;
+  justify-content: center;
+  padding: 18px;
```

Comment:
After playing around with this locally, I think this needs bumped up to something like 40px. It's a bit tight for my taste as it is.